gtk/tests_backend/widgets/canvas.py

```diff
@@ -75,7 +75,7 @@ async def mouse_drag(self, x1, y1, x2, y2):
 
         event = Gdk.Event.new(Gdk.EventType.MOTION_NOTIFY)
         event.button = 1
-        event.state = Gdk.ModifierType.BUTTON1_MASK
+        event.state = Gdk.ModifierType.BUTTON1_MASK | Gdk.ModifierType.MOD2_MASK
```

Comment:
Suggest we add a comment here to clarify what is going on - it won't be obvious otherwise.
```suggestion
        # Add an additional modifier to confirm that keyboard modifiers don't
        # impact event triggering
        event.state = Gdk.ModifierType.BUTTON1_MASK | Gdk.ModifierType.MOD2_MASK
```